### ISSUE TITLE
Random High CPU Frametime

### ISSUE THREAD

**Issue by Emberrss** [qe201020335/HRCounter#13]
While playing any song after configuring HypeRate for Apple Watch, fpsVR shows CPU Frametime spikes to over 10ms and drops a frame every 10-20 seconds. fpsVR history log does not show this weirdly, but I captured a screenshot of the widget right after a spike. 

![ERHJ](https://user-images.githubusercontent.com/76892491/129988823-e1b27b62-4052-4f69-a009-363105de4341.PNG)

**Comment by qe201020335** [comment]
Thanks for the report! I was able to recreate this problem on my own game. The cpu frame time spike occurs every 15 seconds 

HypeRate api requires a heart beat message to enssure a connection. 15 seconds is the interval I set for each heart beat message. I did not bother to send the message async since it is already on a separate thread. 
However, appreantly it still blocks the rendering thread so I will change to async.
